### PR TITLE
Added minimum information required for Casablanca

### LINKED ISSUE #7: Complete Casablanca information
### Definition of Done / Acceptance Criteria

The entry will include other names for the country and at least one thing you can see there.

### Testing

_No response_

### Documentation

_No response_

### Dependencies

_No response_

### Comments

_No response_

### PR DESCRIPTION
**Pull request by hjrucl90**
Closes #7

### CODE REVIEW

**Review by lbeer-ldn — APPROVED** [review]
Looks good!

File: web_source/cities/africa/morocco/casablanca.md [hunk 1]
```diff
@@ -4,12 +4,15 @@ Country: Morocco (Africa)
...
-FIXME
+The White House
 
 ### What we would like to see there
 
-FIXME
+- Royal Palace of Casablanca
+- Sindibad theme park
+- Casablanca Beach
+![Casablanca shoreline](https://upload.wikimedia.org/wikipedia/commons/thumb/4/4a/Bord_de_la_mer_de_ville_casablanca.jpg/1920px-Bord_de_la_mer_de_ville_casablanca.jpg)
```
Comment (hjrucl90) [inline web_source/cities/africa/morocco/casablanca.md:14]: I want to go there so bad!

Comment (arc-teaching) [reply web_source/cities/africa/morocco/casablanca.md:14]: me too!

Comment (lbeer-ldn) [reply web_source/cities/africa/morocco/casablanca.md:14]: fun!

**Pull request merged**